packages/airnode-utilities/src/caching/index.ts

```diff
@@ -65,18 +65,18 @@ const getKeys = (prefix?: string): string[] => {
  * @param key the key whose contents should be returned
  */
 const getValueForKey = (key: string): any | undefined => {
-  try {
-    if (!existsSync(join(CACHE_BASE_PATH, key))) {
-      return undefined;
-    }
+  if (!existsSync(join(CACHE_BASE_PATH, key))) {
+    return undefined;
+  }
 
-    return JSON.parse(readFileSync(join(CACHE_BASE_PATH, key)).toString());
-  } catch (e) {
+  const goRes = goSync(() => readFileSync(join(CACHE_BASE_PATH, key)));
+  if (!goRes.success) {
     logger.error(`Unable to read key data from fs cache`);
-    logger.error((e as Error).stack!);
+    logger.error(goRes.error.stack!);
+    return undefined;
   }
 
-  return undefined;
+  return JSON.parse(goRes.data.toString());
```

Comment:
JSON.parse can throw so this should be inside the `goSync` (similarly how it was previously in try-catch)

Reply:
I see you reverted this in a separate commit. For the future, try to squash it with the previous commit.

Reply:
I'll keep this in mind :+1: